lib/routes/index.js

```diff
@@ -55,14 +86,25 @@ module.exports = (options) => {
 	}
 
 	router.use('/admin/*', jwt);
+	router.use('/respondent/*', jwt);
 
 	// for respondent; TODO: add respondent validation
 	router.get('/tests/:id', async (ctx) => {
 		await quiz.getQuizById(ctx, ctx.params.id);
 	});
 
-	router.get('/activation/:id', async (ctx) => {
-		await activation.getActivationById(ctx, ctx.params.id);
+	router.get('/respondent/activation/:id', async (ctx) => {
+		const foundRespondent = await respondentSchema.findOne({ id: ctx.state.userId.id });
```

Comment:
I think we don't need this check, because jwt do it for us, or move it to separate middleware, because we could have many ```/respondent``` endpoints

Reply:
Ok, it's fine, you need Respondent later here